packages/Notification/Notification.jsx

```diff
@@ -70,31 +70,41 @@ class Notification extends React.Component {
       >
         <FlexGrid>
           <FlexGrid.Row>
-            <FlexGrid.Col xs={dismissible ? 11 : undefined}>
-              <Box inline between={3}>
-                {isImportant(variant) ? renderIcon(iconByVariant[variant]) : undefined}
-                <div
-                  className={joinClassNames(
-                    isImportant(variant) ? styles.withIcon : styles.withoutIcon
+            <FlexGrid.Col>
+              <FlexGrid gutter={false}>
+                <FlexGrid.Row>
+                  <FlexGrid.Col xs={dismissible ? 11 : undefined}>
```

Comment:
Let's see about refactoring this to use a vanilla flexbox that uses normal CSS for spacing so we could avoid this huge tree of FlexGrid components.